oci/client.py

```diff
@@ -942,7 +942,7 @@ def put_blob(
         image_reference: typing.Union[str, om.OciImageReference],
         digest: str,
         octets_count: int,
-        data: requests.models.Response,
+        data: requests.models.Response | typing.Generator | bytes | typing.IO,
```

Comment:
Please use `collections.abc.Generator` instead ([PEP 585](https://peps.python.org/pep-0585/)) 🙃

Reply:
done. I really hate this lengthy package-name, but have to acknowledge pep-585 😤